freetextresponse/settings.py

```diff
@@ -16,4 +16,6 @@
     'freetextresponse',
 )
 
+ALLOWED_HOSTS = ['*']
```

Comment:
According to stackoverflow, need to add this field, which solves the issue. But * should also not be included in production, so I need to figure out what the allowed hosts actually are.